src/components/DataTable/components/Cell/tests/Cell.test.tsx

```diff
@@ -125,34 +119,32 @@ describe('<Cell />', () => {
 
     describe('when set to ascending', () => {
       it('renders an up caret Icon when table is currently sorted by that column', () => {
-        const cell = shallowWithAppProvider(
+        const cell = mountWithTable(
           <Cell header firstColumn sortable sorted sortDirection="ascending" />,
         );
 
         expect(cell.find(Icon).prop('source')).toBe(CaretUpMinor);
       });
 
       it('renders an Icon with an accessibility label indicating the next sort direction is descending', () => {
-        const cell = mountWithAppProvider(
+        const cell = mountWithTable(
           <Cell header firstColumn sortable sorted sortDirection="ascending" />,
         );
 
-        const expectedLabel = cell
-          .instance()
-          .context.polaris.intl.translate(
-            'Polaris.DataTable.sortAccessibilityLabel',
-            {
-              direction: 'descending',
-            },
-          );
+        const expectedLabel = cell.app.intl.translate(
```

Comment:
I added this to the testing utilities previously so we can access the translations

Reply:
Reading some of the Slack history in shopify-web the proposed way of testing translations is to check for the english text rather than trying to hook into translations stuff.

I think that makes sense as given this current content if we were to goof and make the `sortAccessibilityLabel` translation empty then this test would still pass as you're basically comparing a variable to itself

Reply:
Makes sense to me, change these tests 👍